ports/lpc55/boards.c

```diff
@@ -123,34 +113,44 @@ void board_dfu_complete(void)
   NVIC_SystemReset();
 }
 
+/* Defined in board_flash.c 
```

Comment:
would you mind telling why this function can't be implemented here ?

Reply:
Reading erased flash causes a bus fault due to ecc error.  You need to use FLASH_Read() to check.  I moved this to board_flash.c so that all the fsl_iap routines are together.  That is why I put board_usb_get_serial  there as well since FFR_GetUUID is in fsl_iap_ffr.h  I could add the headers to boards.c if that makes more sense.

Reply:
thank you, that make sense, no problem at all. We can refactor this later on, I just wonder why would you move it.